vue.config.js

```diff
@@ -47,6 +50,24 @@ module.exports = {
       return args;
     })
     .end()
+    .plugin('define')
+    .tap((options) => {
+      const definitions = { ...options[0] };
+
+      Object.entries(definitions['process.env']).forEach(([k, v]) => {
+        definitions[`process.env.${k}`] = v;
+      });
+      delete definitions['process.env'];
+
+      // eslint-disable-next-line camelcase
+      if (npm_package_version) {
+        definitions['process.env.npm_package_version'] = JSON.stringify(npm_package_version);
+      }
+      const gitRevisionPlugin = new GitRevisionPlugin();
```

Comment:
Actually, it is not used as a webpack plugin, maybe we can use something super simple like https://stackoverflow.com/a/35778030/6176994? (but to put require at the top of the file)